### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by alistairgd** [Rem0o/FanControl.Releases#1234] [General]
So my new PC build was annoying me which is not a good feeling for a new gaming rig.

Mostly my RTX3070 was reaching 83 degrees while gaming, so I added a 4th case fan but the noise of all these fans when not gaming was annoying. I googled for an app that would let me control case fans depending on **GPU** temps and found nothing.

Then I found this app in a YT video - problem solved! My extra fan now has a curve that is set to zero until the GPU reaches 65 degrees then the fan kicks in to 100% - perfect.

**Comment by Rem0o** [comment]
A classic use-case that is totally ignored by manufacturers. Exact reason why I started to develop FanControl in the first place.

**Reply by alistairgd** [reply]
Here's something else that I think is largely ignored by manufacturers.

Considering that the GPU is the biggest source of heat when gaming - and the exhaust ports sometimes(?) point straight to the case side window where the heat bounces back towards the card - there should be a way of having an exhaust fan right there on the side window/panel that powers up with the GPU on load and sucks out that hot air, and sits idle and quiet at other times.
